src/main/java/me/moodcat/backend/rooms/RoomInstance.java

```diff
@@ -273,10 +276,24 @@ public ChatMessageModel sendMessage(final ChatMessageModel model,
         return model;
     }
 
+    private void verifyNonSpamming(final User user) {
+        final long currentTime = System.currentTimeMillis();
```

Comment:
What happened with the System check? This has to be implemented for the EndToEndTests and general testing. Also with System messages they should not be filtered.